lib/cmd/fh3/app/_baseCloudRequest.js

```diff
@@ -4,7 +4,7 @@ var read = require('../../fh2/app/read.js').read;
 var log = require("../../../utils/log");
 var util = require('util');
 var fhc = require("../../../fhc");
-var request = require('request').defaults({'proxy': fhc.config.get("proxy")});
+var request = require('request');     // don't set proxy or other configured defaults here, since the config may not be loaded when this module is
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)